mtutest unit test fails when vermont build with DISABLE_UDP_CONNECT=ON

The mtutest unit test fails if vermont is built with DISABLE_UDP_CONNECT=ON

```
mkdir build && cd build
cmake -DDISABLE_UDP_CONNECT=ON ../
make
make test

Running tests...
Test project /home/brownn/work/DANOS/vermont/build
    Start 1: vermonttest_build
1/4 Test #1: vermonttest_build ................   Passed    0.24 sec
    Start 2: vermont
2/4 Test #2: vermont ..........................   Passed    9.58 sec
    Start 3: example_2
3/4 Test #3: example_2 ........................   Passed    0.01 sec
    Start 4: mtutest
4/4 Test #4: mtutest ..........................***Failed    0.00 sec

75% tests passed, 1 tests failed out of 4

Total Test time (real) =   9.84 sec

The following tests FAILED:
	  4 - mtutest (Failed)
Errors while running CTest
make: *** [Makefile:98: test] Error 8
```

```
$ ./mtutest
03/03 19:32:34 CRITICAL: getsockopt(...,IP_MTU,...) failed, Transport endpoint is not connected
ipfix_add_collector() failed.
```

--- a/src/tests/ipfixlolib/CMakeLists.txt
+++ b/src/tests/ipfixlolib/CMakeLists.txt
@@ -56,4 +56,6 @@ IF (JOURNALD_FOUND)
 ENDIF (JOURNALD_FOUND)
 
 ADD_TEST(example_2 example_code_2)
-ADD_TEST(mtutest mtutest)
+IF (NOT DISABLE_UDP_CONNECT)
+	ADD_TEST(mtutest mtutest)
+ENDIF (NOT DISABLE_UDP_CONNECT)

